test/unit/api_spec.js

```diff
@@ -405,6 +403,12 @@ describe('api', function() {
         done.fail(reason);
       });
     });
+    it('gets current workerSrc', function(done) {
+      let workerSrc = PDFWorker.getWorkerSrc();
+      expect(typeof workerSrc).toEqual('string');
+      expect(workerSrc).toEqual(PDFJS.workerSrc);
+      done();
```

Comment:
Since the test is synchronous, remove `done();` and `done` in the `function(done){`. `done` is only needed for async tests.

Reply:
Done :-)